src/olympia/addons/admin.py

```diff
@@ -83,8 +85,13 @@ def clean(self):
         path = None
         try:
             path = self.data.get('path')
-            path = ('/' if not path.startswith('/') else '') + path
-            resolve(path)
+            site = settings.SITE_URL
+            if (models.ReplacementAddon.is_external_path(path) and
+                    not path.startswith(site)):
```

Comment:
/me doesn't love the indentation 🙈

Reply:
that's the curse of `if (` being 4 characters long